phpBB/phpbb/extension/metadata_manager.php

```diff
@@ -229,7 +229,10 @@ public function validate($name = 'display')
 			case 'all':
 				$this->validate('display');
 
-				$this->validate_enable();
+				if (!$this->validate_enable())
+				{
+					throw new \phpbb\extension\exception($this->user->lang('META_FIELD_NOT_SET', $name));
```

Comment:
That doesn't make much sense right now? It will just say:
`Required meta field all has not been set.`